src/components/ActionCard.tsx

```diff
@@ -92,23 +51,62 @@ const ActionCard: React.FC<IProps> = ({
   const isImageURL = typeof image === "string"
 
   return (
-    <Card to={to} className={className} hideArrow={true}>
-      <ImageWrapper
-        isRight={isRight}
-        isBottom={isBottom}
+    <LinkBox
+      boxShadow="
+	  0px 14px 66px rgba(0, 0, 0, 0.07),
+    0px 10px 17px rgba(0, 0, 0, 0.03), 0px 4px 7px rgba(0, 0, 0, 0.05)"
+      color="text"
+      flex="1 1 372px"
+      _hover={linkBoxFocusStyles}
+      _focus={linkBoxFocusStyles}
+      className={className}
+      m={4}
+    >
+      <Flex
+        minH="260px"
+        bg="cardGradient"
+        direction="row"
+        justify={isRight ? "flex-end" : "center"}
+        align={isBottom ? "flex-end" : "center"}
         className="action-card-image-wrapper"
+        boxShadow="inset 0px -1px 0px rgba(0, 0, 0, 0.1)"
       >
-        {!isImageURL && <Image image={image} alt={alt || ""} />}
+        {!isImageURL && (
+          <Image
+            alt={alt || ""}
+            as={GatsbyImage}
+            maxH="257px"
+            maxW={{ base: "372px", sm: "311px" }}
```

Comment:
Okay, as suggested, I am using Image. The breakpoint thing was nice to know about. Thank you for [mentioning that in the Migration Guide.](https://github.com/ethereum/ethereum-org-website/blob/dev/docs/chakra-migration-guide.md#breakpoints) But, just to inform you the example the in Migration Guide is inversed.

![image](https://user-images.githubusercontent.com/51415616/200138887-ec675bd3-a23d-4946-97b1-de2e41b64a38.png)

Reply:
Ahh, okay. Thank you for explaining that.